### PR TITLE
refined log ;configured whitelabel; Added ApiErrorController;Integration...

### PR DESCRIPTION
**Pull request by DhivyaRamasamy1**
... test using RestAssured;Pom file Maven dependency updated

### CODE REVIEW

**Comment by DhivyaRamasamy1** [comment]
Hi @fiallega ,
Kindly review my branch and feedback

File: src/main/java/gov/miamidade/wasd/controller/ApiController.java [hunk 1]
```diff
@@ -13,7 +14,7 @@
  */
 
 @RestController
-@RequestMapping(value="/api")
+@RequestMapping(value="/")
```
Comment (fiallega) [inline src/main/java/gov/miamidade/wasd/controller/ApiController.java]: why did you remove the /api path. I thought we had agreed on that

File: src/main/java/gov/miamidade/wasd/controller/ApiErrorController.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * Created by dhivya on 2/10/2015.
+ */
+@RestController
+public class ApiErrorController implements ErrorController{
+
+    private static final String ERROR_PATH = "/error";
+
+    @RequestMapping(value=ERROR_PATH)
+    public String handleError(){
+       System.out.println("ApiErrorController :handleError");
```
Comment (fiallega) [inline src/main/java/gov/miamidade/wasd/controller/ApiErrorController.java]: can you log instead of System.out

**Comment by fiallega** [comment]
@DhivyaRamasamy1 look at the comments below.
Once you fix them you can merge the pull request and delete the branch
